pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -205,7 +205,12 @@ sub _suggest_kernelcare {
         }
 
         $suggestion = ($suggestion) ? '<p/><p/>' . $suggestion : '';
-        $promotion = $self->_lh->maketext('KernelCare provides an easy and effortless way to ensure that your operating system uses the most up-to-date kernel without the need to reboot your server. After you purchase and install KernelCare, you can obtain and install the KernelCare "Extra" Patchset, which includes symlink protection.');
+        $promotion = $self->_lh->maketext('KernelCare provides an easy and effortless way to ensure that your operating system uses the most up-to-date kernel without the need to reboot your server.');
+
+        # Verifies the user is on CentOS 6 or 7, and is not running CloudLinux.
+        if ( Cpanel::KernelCare::system_supports_kernelcare_free() ) {
```

Comment:
❗️ This function does not exist in v70, which will still be supported until March 30, IIRC.  The function does exist in v76 and later.

We should add a check for the version, in case this (or a later Security Advisor change) needs to be backported to v70.

Reply:
The only changes we will make in v70 before it goes EOL is security updates or otherwise critical upcp fixes.. I don't disagree it is incorrect, but this isn't likely to be an issue.

Reply:
Understood.  Please keep this in mind until March 30th.